docs/UserGuide.md

```diff
@@ -319,7 +347,7 @@ Restrictions:
 
 Examples:
 * `ecreate n/clean beach m/10 trash bags ...` (refer to [ecreate](#creating-an-event-ecreate) above for full command) creates an 
-  event `clean beach` that requires `20 trash bags`. By performing `efind clean beach`, then `eaddm eid/1 10 trash bags`,
+  event `clean beach` that requires `20 trash bags`. By performing `efind clean beach`, then `eaddm eid/1 m/10 trash bags`,
```

Comment:
small issue but should be `efind n/clean beach` instead

Reply:
noted with thanks :D